marketplacetests/marketplace/app.py

```diff
@@ -104,6 +104,7 @@ def search(self, term):
 
     def set_region(self, region):
         # go to the :debug page
+        self.wait_for_element_displayed(*self._search_locator)
```

Comment:
Wouldn't we also want this in the `search` method? Also, shouldn't `send_keys` fail if the element is not displayed?